harmony/cli.py

```diff
@@ -65,6 +65,8 @@ def setup_cli(parser):
     parser.add_argument('--harmony-input',
                         help=('the input data for the action provided by Harmony, required for '
                               '--harmony-action=invoke'))
+    parser.add_argument('--harmony-input-file',
+                        help=('the optional path to the input data for the action provided by Harmony, required for '))
```

Comment:
missing `'--harmony-action=invoke'`. May also want to change the `required for` wording.

Reply:
actually I meant to delete that part altogether, as it it not required